src/igr-tests/Makefile

```diff
@@ -1,16 +1,13 @@
 include ../../mconfig
 
 check-igr: igr-runner
-	DINIT_EXEC=../../dinit DINITCTL_EXEC=../../dinitctl DINITCHECK_EXEC=../../dinitcheck DINITMONITOR_EXEC=../../dinit-monitor ./igr-runner
+	IGR_FUNCTIONS="../igr_functions" EXEC_PREFIX="../../" MAKE=1 ./igr-runner
```

Comment:
Doesn't look like "MAKE=1" is needed, can we remove it?

Reply:
Ok I see where it is used (in igr_functions) but I think it should be removed, see other comment.